caffe2/proto/torch.proto

```diff
@@ -23,7 +28,7 @@ message TensorDef {
 
   optional RecordRef data = 6;
 
-  // future: device options
+  optional DeviceDef device = 7;
```

Comment:
Maybe we should store device as its device string. Why?

1. c10::Device already includes a string constructor and a << form, that will presumably be kept up to date.
2. We will likely continue to support the device string even if we update what can be in it, meaning that import/export code will not need to change even if we change the meaning of device.